packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -148,6 +159,37 @@ const navigationStory = (
   </>
 );
 
+const hamburgerStory = () => {
+  return (
+    <BackgroundContainer>
+      <CanonicalHamburgerMenu announcedText="Menu" onOpen={() => {}} />
+    </BackgroundContainer>
+  );
+};
```

Comment:
Are we planning to reuse and include it as part of another story?

Reply:
Do you mean including it as a part of the Navigation stories with brand?

Reply:
Yes.

Reply:
IMO it does not bring too much value to have a Brand+HamburgerMenu story for now. After this PR or SwipeableNav PR is merged we could add stories for Brand+HamburgerMenu+SwipeableNav

Reply:
I agree, but i was thinking more in terms of. This cannot be reused since it wraps `CannonicalHamburgerMenu` in a `BackgroundContainer`, therefore it would make more sense to put in inside `.add('Hamburger', () => {here})` instead of creating a function and then passing it in.

Reply:
Oh okay, understood. That is a good idea, will fix it